operations/finance.md

```diff
@@ -426,6 +426,8 @@ Returns all accounting items of the enterprise that were consumed \(posted\) or
 | `GrossValue` | number | optional | Gross value including all taxes. |
 | `TaxValues` | array of [Tax Value](finance.md#tax-value) | optional | The tax values applied. |
 
+For most amounts, precision of values depends on [Currency](configuration.md#currency) precision or Tax precision, based on [Pricing](configuration.md#pricing) of the enterprise. But in some cases, precision can be higher.
```

Comment:
Tax precision isn't serialized in Connector, how should partners know what that is. We need to extend `TaxEnvirnment` first so partners can get the information.